desktop/cmp/panel/PanelModel.js

```diff
@@ -68,8 +68,8 @@ export class PanelModel {
      * @param {boolean} [c.resizable] - Can panel be resized?
      * @param {boolean} [c.resizeWhileDragging] - Redraw panel as resize happens?
      * @param {boolean} [c.collapsible] - Can panel be collapsed, showing only its header?
-     * @param {number} c.defaultSize - Default size (in px) of the panel.
-     * @param {number} [c.minSize] - Minimum size (in px) to which the panel can be resized.
+     * @param {(number|string)} config.defaultSize - Default size (in px or %) of the panel.  eg: 300 or '50%'
+     * @param {?number} [c.minSize] - Minimum size (in px) to which the panel can be resized.
```

Comment:
Does minSize / maxSize support % values? The PR title says so, but these comments still say px?

Reply:
No, they do not.  I have corrected the title.